src/Control/Distributed/Spark/RDD.hs

```diff
@@ -112,6 +163,17 @@ textFile sc path = do
   jpath <- reflect (Text.pack path)
   call sc "textFile" [coerce jpath]
 
+-- recordLength = number of bytes
+binaryRecords :: SparkContext -> FilePath -> Int32 -> IO (RDD ByteString)
+binaryRecords sc fp recordLength = do
+  jpath <- reflect (Text.pack fp)
+  call sc "binaryRecords" [coerce jpath, coerce recordLength]
+
+-- | Reinterprets an @RDD a@ as an @RDD b@ provided that both types have
+-- the same representation on the java side.
+rddCoerce :: (Interp a ~ Interp b) => RDD a -> RDD b
```

Comment:
Do we really need this one? If we define this on RDD's then why not on any Java object newtype wrapper? Isn't this simply a case of calling `Data.Coerce.coerce` directly at the call site? Note that `RDD` is not abstract.